### ISSUE TITLE
How to run LSD-SLAM on streaming video?

### ISSUE THREAD

**Issue by st117752** [tum-vision/lsd_slam#209]
Is there any possible way to process video from VLC which can access by using IP address like "rtsp://10.10.10.101:8554/"

Thank you in advance :) :+1:

**Comment by af-silva** [comment]
Yes, first the LSD-SLAM access the video input from a ROS topic, subscribing one like _image_raw_ : 

> /image:=/camera/image_raw 

You can configure this in the launch file of the LSD-SLAM_CORE node.

So if you want to access a video stream using RTSP you first need a ROS node that access this stream and publishes to a topic. Please refer to this link [Video_stream_opencv](http://wiki.ros.org/video_stream_opencv). After this you will have the LSD-SLAM running on the stream, be aware that you have to calibrate the camera and provide the calibration information to the LSD-SLAM if you want to obtained good results.
